### PR TITLE
Update links & info about control panel for Cloud DNS: INFODEV-2693

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4e58b755a6/how-to/)

**Comment by RackerKate** [comment]
@stephamon For your review please, per Catherine.

File: content/cloud-dns/create-a-dkim-txt-record.md [hunk 1]
```diff
@@ -19,61 +19,70 @@ through cryptographic authentication.
...
 
-Before you perform this procedure, choose a text string to use as your selector and
-generate a public-private key pair
+Before you perform this procedure, choose a text string to use as your
+selector and generate a public-private key pair.
 
+1.  Log in to the [Cloud Control Panel](https://login.rackspace.com/).
 
-1.  Log in to the [Cloud Control Panel](https://mycloud.rackspace.com/).
+2.  Click the **Select a Product** drop-down menu in the top navigation bar,
```
Comment (stephamon) [inline content/cloud-dns/create-a-dkim-txt-record.md]: Suggestion: "In the top navigation bar, click **Select a Product*, and then select **Rackspace Cloud**."

If you like that, update to match in the other files :)

Comment (stephamon) [reply content/cloud-dns/create-a-dkim-txt-record.md]: Hm, I feel like this is cleaner, but do you think it sounds strange to have two steps in a row say "In the top navigation bar..."?

File: content/cloud-dns/creating-mx-records-for-google-apps-through-the-cloud-control-panel.md [hunk 2]
```diff
@@ -1,39 +1,44 @@
...
 ---
 
-This article shows how to add [Google Apps](http://www.google.com/enterprise/apps/business/pricing.html) MX records to a domain managed via the Cloud Control Panel.
+This article shows you how to add [Google Apps](http://www.google.com/enterprise/apps/business/pricing.html) MX records to a domain managed via the Cloud Control Panel.
 
 **Important:** Be sure to verify your domain with Google Apps through their administrative interface.
 
-After your domain has been verified, the next step is to add Google's MX records to the domain's information in the DNS
-section of the [Cloud Control Panel](https://mycloud.rackspace.com/).
+After your domain has been verified, the next step is to add Google's MX records to the domain's information in the Domain Name Service (**DNS**)
```
Comment (stephamon) [inline content/cloud-dns/creating-mx-records-for-google-apps-through-the-cloud-control-panel.md]: (DNS) doesn't need to be bold here.

File: content/cloud-dns/choosing-dns-record-types.md [hunk 3]
```diff
@@ -5,15 +5,16 @@ title: Choosing DNS record types
...
+last_modified_by: Kate Dougherty
 product: Cloud DNS
 product_url: cloud-dns
 ---
 
 DNS uses different record types for different purposes. This
 article helps you determine which record type you should use when
-creating a DNS record in the Cloud Control Panel.
+creating a DNS record in the [Cloud Control
+Panel](http://login.rackspace.com/).
```
Comment (stephamon) [inline content/cloud-dns/choosing-dns-record-types.md]: Make sure you're using `https://` and not `http://` for the log in URLs.

**Comment by stephamon** [comment]
Just a few things in comments @RackerKate

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0bcc3a6367/how-to/).

**Comment by RackerKate** [comment]
@cyrichardson Please review - OK to go.

**Comment by cyrichardson** [comment]
Thanks, @RackerKate. On it.

File: content/cloud-dns/create-a-reverse-dns-record.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ title: Create a reverse DNS record
...
 product: Cloud DNS
 product_url: cloud-dns
 ---
 
-Reverse DNS records are essential for those running a mail server since
-many recipient servers reject, or mark as spam, all email that
-originates from an "unauthenticated" server.
+Reverse Domain Name Service (DNS) records are essential for those running a
+mail server because many recipient servers reject, or mark as spam, all email
+that originates from an "unauthenticated" server.
```
Comment (cyrichardson) [inline content/cloud-dns/create-a-reverse-dns-record.md]: an unauthenticated server

File: content/cloud-dns/create-a-reverse-dns-record.md [hunk 5]
```diff
@@ -31,34 +31,35 @@ RDNS can also be very useful when tracking down network issues and was
...
-When you enter a domain name into your browser, the DNS system will find
+When you enter a domain name into your browser, the DNS system finds
 the IP address of the server the domain is associated with.
 
 A reverse DNS lookup does the opposite. It establishes what domain is
-associated with the IP address. This is a useful setting to configure
-for anyone, but it is essential for customers running an outgoing mail
-server on their Cloud Server.
+associated with the IP address. This is a useful setting to configure, and
+essential for customers who run an outgoing mail server on their
```
Comment (cyrichardson) [inline content/cloud-dns/create-a-reverse-dns-record.md]: is essential for

File: content/cloud-dns/create-a-reverse-dns-record.md [hunk 6]
```diff
@@ -31,34 +31,35 @@ RDNS can also be very useful when tracking down network issues and was
...
 
-When you enter a domain name into your browser, the DNS system will find
+When you enter a domain name into your browser, the DNS system finds
 the IP address of the server the domain is associated with.
 
 A reverse DNS lookup does the opposite. It establishes what domain is
-associated with the IP address. This is a useful setting to configure
-for anyone, but it is essential for customers running an outgoing mail
-server on their Cloud Server.
+associated with the IP address. This is a useful setting to configure, and
```
Comment (cyrichardson) [inline content/cloud-dns/create-a-reverse-dns-record.md]: configure and

**Comment by RackerKate** [comment]
@cyrichardson Please review with new wording.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b26b4f1d97/how-to/)

File: content/cloud-dns/create-an-spf-txt-record.md [hunk 7]
```diff
@@ -89,24 +89,26 @@ following basic markers:
 To add an SPF TXT record by using the Cloud Control Panel, use the following
 steps:
```
Comment (cyrichardson) [inline content/cloud-dns/create-an-spf-txt-record.md]: Italicized here through end of file, but looks OK in view mode.

File: content/cloud-dns/index.md [hunk 8]
```diff
@@ -3,16 +3,16 @@ title: Cloud DNS
...
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2018-10-23'
+last_modified_by: Kate Dougherty
 permalink: /cloud-dns/
 product: Cloud DNS
 product_url: cloud-dns
 ---
 
-The Domain Name System is essentially the phonebook for the Internet. It manages the relationships between IP addresses (the phone numbers), and domains (individuals and corporations). All domains and their components, such as mail servers, utilize DNS to resolve to the appropriate locations.
+The Domain Name System (DNS) is essentially the phonebook for the Internet. It manages the relationships between IP addresses (the phone numbers), and domains (individuals and corporations). All domains and their components, such as mail servers, use DNS to resolve to the appropriate locations.
```
Comment (cyrichardson) [inline content/cloud-dns/index.md]: (the phone numbers) and domains

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f0892506c3/how-to/).

**Pull request merged**